docs/data/rpc/api-reference/structure/data-format.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 60
+title: Data Format
+---
+
+import { ExampleResponse } from "@site/src/components/ExampleResponse";
+import { AttributeTable } from "@site/src/components/AttributeTable";
+
+### XDR Format
+
+In the Stellar network, transactions are encoded using a standardized protocol called [External Data Representation](https://en.wikipedia.org/wiki/External_Data_Representation) (XDR).
+
+In RPC, you will encounter XDR when [simulating](../methods/simulateTransaction) and [sending](../methods/sendTransaction) transactions, as well as when retrieving [transactions](../methods/getTransactions), [ledgers](../methods/getLedgers), and [ledger entries](../methods/getLedgerEntries).
+
+By default, RPC will return all XDR attributes as the machine-readable base64 encoded XDR string. All XDR encoded response fields are suffixed with `Xdr`. You can decode this XDR in the Stellar Lab’s [XDR page](https://lab.stellar.org/xdr/view?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;).
```

Comment:
unfortunately, not all:

```suggestion
By default, RPC will return all XDR attributes as the machine-readable base64 encoded XDR string. XDR-encoded response fields are usually suffixed with `Xdr`. You can decode this XDR in the Stellar Lab’s [XDR page](https://lab.stellar.org/xdr/view?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;).
```

Reply:
Dang it all! Thanks, which are not?

Reply:
https://developers.stellar.org/docs/data/rpc/api-reference/methods/getLedgerEntries is the biggest offender :/ `xdr` and `key` for the `xdr.LedgerEntryData` and `xdr.LedgerKey`, respectively (awesome right)